index.js

```diff
@@ -5,6 +5,8 @@ import { patch, unpatchAll } from '@vizality/patcher';
 import { getModule } from '@vizality/webpack';
 import { ContextMenu } from '@vizality/components';
 
+const githubRegex = /https?:\/\/github\.com\/[a-zA-Z0-9-]+\/[a-zA-Z0-9-]+\/?$/m;
```

Comment:
Only referenced once, does not need to be defined as a constant.

Reply:
I suppose as a defined asvariable is more suitable?

Reply:
Also esinit prefers it to be defined as a constant

Reply:
Done as per 8387a2bed1d443973701fb2e7406733d8e08dfd8